CONTRIBUTING.md

```diff
@@ -19,8 +19,39 @@ limitations under the License.
 # Contributing
 
 Welcome to Elyra! If you are interested in contributing to the [Elyra code repo](README.md)
-then checkout the [Contributor's Guide](https://github.com/elyra-ai/community/blob/master/contributing.md).
+then checkout the [Contributor's Guide](https://github.com/elyra-ai/community/blob/master/contributing.md). 
 
 The [Elyra community repo](https://github.com/elyra-ai/community) contains information on how the community
 is organized and other information that is pertinent to contributing.
 
+### Getting Started
+
+Before contributing, we suggest reading the [Code of Conduct](https://github.com/elyra-ai/community/blob/master/code-of-conduct.md) & 
+[Contributor Repo Setup Documentation](https://elyra.readthedocs.io/en/latest/developer_guide/documentation.html). 
+Once completed, it's encouraged that you look under the [Issues](https://github.com/elyra-ai/elyra/issues) tab for contribution opportunites.
+For those new to the project or OpenSource workflows, we suggest looking under:
+
+- Issues tagged as [Good First Use](https://github.com/elyra-ai/elyra/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22)
+- Issues tagged as [Help Wanted](https://github.com/elyra-ai/elyra/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22 )
+
+If you are planning to contribute to source code, the following can help you set up your [development environment](https://elyra.readthedocs.io/en/latest/developer_guide/development-workflow.html).
+
+
+#### Suggested Small to Medium Sized Projects
+
+- Add Support for R Script Editor
+- Seamless Integration with S3/COS from Elyra UI (Integrated Browser)
+- Implement Debug Support for IPython
+- Better Support for Data Processing
+- Data Sources Extension, SQL Editor
+- Property / Metadata Inheritance / Scopes
```

Comment:
Since these items are more like a roadmap and can change often, I would suggest removing these lines. The link to `Good first project` issues will already list them, and this doc wouldn't have to be updated whenever a new issue with that tag is added/removed.
Then this section can be merged within `Getting Started` above (remove the `Suggested Small to Medium Sized Projects` title).

Reply:
I completely agree. The `Suggested Small to Medium Sized Projects` would inevitably be obsolete (assuming the projects get pursued). I condensed the two sections. Resolved.